### PR TITLE
docs: Added explore Traces plugin to Tempo Quick start tutorial

### PR DESCRIPTION
**Pull request by Jayclifford345**
This PR modifies the core quick start demo to include the Explore Traces plugin. I opted to also enable metrics from spans as well as fun traces so we could generate some Error to complete the RED view.

I have also added a section to the quick start docs guide to showcase a scenario of using the plugin to identify why some spans have particularly long duration times compared to the median.

I have also updated the Killercoda dev environment for testing: https://killercoda.com/grafana-dev-testing/course/tempo/quick-start

Note that the Trace Explore plugin will not appear in the dev environment since it realises on pulling the docker-compose from our main branch. If you would like to test copy the docker-compose changes and the tempo config changes from this PR into the VM before you spin up the docker-compose file.

**Checklist**
- [ ] Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by javiermolinar — APPROVED** [review]

File: docs/sources/tempo/getting-started/docker-example.md [hunk 1]
```diff
@@ -117,6 +117,44 @@ You can use Grafana to explore the traces generated by the k6-tracing service.
     
 <!-- INTERACTIVE page step2.md END -->
 
+<!-- INTERACTIVE page step3.md START -->
+
+## Explore Traces Plugin
```
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/docker-example.md:122]: Using sentence case
```suggestion
## Explore Traces plugin
```

File: docs/sources/tempo/getting-started/docker-example.md [hunk 2]
```diff
@@ -117,6 +117,44 @@ You can use Grafana to explore the traces generated by the k6-tracing service.
...
+
+Another method to explore traces within Grafana is via the [Explore Traces](https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/) plugin. This plugin offers an opinionated non query-based approach to exploring traces. Lets take a look at some of its key features and panels.
```
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/docker-example.md:124]: Any sugestions for a different word besides "explore traces using Explore Traces"
```suggestion
Another method to explore traces within Grafana is using the [Explore Traces](https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/) plugin. This plugin offers an opinionated non query-based approach to exploring traces. Lets take a look at some of its key features and panels. 
```

File: docs/sources/tempo/getting-started/docker-example.md [hunk 3]
```diff
@@ -117,6 +117,44 @@ You can use Grafana to explore the traces generated by the k6-tracing service.
...
 <!-- INTERACTIVE page step2.md END -->
 
+<!-- INTERACTIVE page step3.md START -->
+
+## Explore Traces Plugin
+
+Another method to explore traces within Grafana is via the [Explore Traces](https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/) plugin. This plugin offers an opinionated non query-based approach to exploring traces. Lets take a look at some of its key features and panels. 
+
+1. Open a browser and navigate to [http://localhost:3000/a/grafana-exploretraces-app](http://localhost:3000/a/grafana-exploretraces-app).
+2. Within the filter bar there is a dropdown menu set to `Rate` of `Full traces`. Change this to `Duration` and `All spans`.
```
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/docker-example.md:127]:
```suggestion
2. Within the filter bar, there is a dropdown menu set to **Rate** of **Full traces**. Change this to **Duration** and **All spans**.
```

File: docs/sources/tempo/getting-started/docker-example.md [hunk 4]
```diff
@@ -117,6 +117,44 @@ You can use Grafana to explore the traces generated by the k6-tracing service.
...
+
+This should provide an updated panel view like this.
```
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/docker-example.md:129]:
```suggestion
This updated panel view looks like this:
```

File: docs/sources/tempo/getting-started/docker-example.md [hunk 5]
```diff
@@ -117,6 +117,44 @@ You can use Grafana to explore the traces generated by the k6-tracing service.
...
+
+{{< figure align="center" src="/media/docs/tempo/explore-spans-error-view.png" alt="Explore Traces panel" >}}
+
+Breakdown of the view:
+* The histogram at the top shows the distribution of span durations. The lighter the color, the more spans in that duration bucket. In this example most spans fall within `537ms` which is considered the average duration for the system.
```
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/docker-example.md:134]:
```suggestion
* The histogram at the top shows the distribution of span durations. The lighter the color, the more spans in that duration bucket. In this example, most spans fall within `537ms`, which is considered the average duration for the system.
```

File: docs/sources/tempo/getting-started/docker-example.md [hunk 6]
```diff
@@ -117,6 +117,44 @@ You can use Grafana to explore the traces generated by the k6-tracing service.
...
+* There are some high peaks in the histogram, which indicate spans that are taking longer than the average (As high as `2.15s`). These are likely to be the spans that are causing performance issues. Lets investigate further to see if we can identify the root cause.
```
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/docker-example.md:135]:
```suggestion
* The high peaks in the histogram indicate spans that are taking longer than the average (As high as `2.15s`). These are likely to be the spans that are causing performance issues. Let's investigate further to see if we can identify the root cause.
```

File: docs/sources/tempo/getting-started/docker-example.md [hunk 7]
```diff
@@ -117,6 +117,44 @@ You can use Grafana to explore the traces generated by the k6-tracing service.
...
+`shop-backend` appears to be the primary culprit for the slow traces. This happens when a user initiates the `article-to-cart` operation. From here we can select the Trace Name which will open the Trace View panel. 
+
+{{< figure align="center" src="/media/docs/tempo/slow-trace-trace-view.png" alt="Trace View panel" >}}
+
+The Trace View panel provides a detailed view of the trace. The panel is divided into three sections:
+* The top section shows the trace ID, duration, and the service that generated the trace.
+* The middle section shows the trace timeline. Each span is represented as a horizontal bar. The color of the bar represents the span's status. The width of the bar represents the duration of the span.
+* The bottom section shows the details of the selected span. This includes the span name, duration, and tags.
+
+Drilling into the `shop-backend` span, we can see that the `place-articles` operation has an exception event tied to it. This is likely the root cause of the slow trace.
```
Comment (knylander-grafana) [inline docs/sources/tempo/getting-started/docker-example.md:150]:
```suggestion
Drilling into the `shop-backend` span, you can see that the `place-articles` operation has an exception event tied to it. This is likely the root cause of the slow trace. 
```

**Review by knylander-grafana — APPROVED** [review]
Looks great! Thank you for doing this. I have some minor suggestions, mostly stylistic.

**Comment by Jayclifford345** [comment]
Amazing! Thanks Kim! That's all the changes accepted and I changed the sentence by removing the first line.

**Review by knylander-grafana — APPROVED** [review]

**Pull request merged**